### ISSUE TITLE
Keras version 2.1.2 bump

### ISSUE THREAD

**Issue by hgaiser** [fchollet/keras#8650]
Due to the error introduced in https://github.com/fchollet/keras/pull/8056 , custom loss functions with mismatching shapes for `y_true` and `y_pred` resulted in an exception being thrown. This issue is fixed in https://github.com/fchollet/keras/pull/8514 . However this fix is not present in the latest Keras release, so I'm forced to use `master` or `2.0.9`.

Eventhough it is in the README.md and a clear issue is open for this, I see roughly every day someone asking about this error on https://github.com/fizyr/keras-retinanet . My request is if there can be a bump in Keras version so that we can tell people to update to the latest Keras.

**Comment by fchollet** [comment]
Yes, in fact 2.1.2 is planned for today. We'll typically release every 2 weeks going forward.

**Comment by hgaiser** [comment]
Good news, thanks!

**Comment by fchollet** [comment]
Now released.

**Comment by hgaiser** [comment]
Thanks for the heads up! I'll update our readme.

On Dec 1, 2017 7:32 PM, "François Chollet" <notifications@github.com> wrote:

Now released.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/fchollet/keras/issues/8650#issuecomment-348572242>, or mute
the thread
